client/components/select-dropdown/test/index.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+import { assert } from 'chai';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import SelectDropdown from '../index';
+
+describe( 'index', function() {
+	require( 'test/helpers/use-fake-dom' )();
```

Comment:
Sure. In some cases when we pull in these dependencies they can interact with global data and mess with each other. I don't think there's something specific here with `useFakeDom()`, but maybe @gwwar was trying to figure out why it was a `require()` in-place instead of an `import` at the top level. I cannot speak for her though.